vtex/faststore
# Feat: Third Party API Schema extensions

## What's the purpose of this pull request?

Add thirdParty api extension feature.

## How it works?
Basing on [vtex api extension PR](https://github.com/vtex/faststore/pull/1913), the third party api extension feature works using the `packages/core/src/customizations/graphql/thirdParty` folder

## How to test it?

- run packages/core with `yarn dev`
-  execute this curl in a client of yor preference

```
curl --request POST \
  --url http://localhost:3000/api/graphql \
  --header 'Content-Type: application/json' \
  --data '{"query":"query Data {\n\textraData {\n\t\tdata\n\t}\n\tnamedExtraData(name: \"Hello\") {\n\t\tdata\n\t}\n}\n","operationName":"Data"}'
```
- the returnal should be like:
```
{"data":{"extraData":{"data":"Extra data"},"namedExtraData":{"data":"Named extra data: Hello"}}}
```
### Starters Deploy Preview





## References
